.github/workflows/build_publish.yml

```diff
@@ -77,14 +83,27 @@ jobs:
         env:
           SERVICE_VERSION: ${{ steps.tag.outputs.tag }}
 
-#  report-to-sherlock:
-#    # Report new Sam version to Broad DevOps
-#    uses: broadinstitute/sherlock/.github/workflows/client-report-app-version.yaml@main
-#    needs: [ cda-service-build-tag-publish ]
-#    with:
-#      new-version: ${{ needs.cda-service-build-tag-publish.outputs.tag }}
-#      chart-name: 'cancerdata'
-#    permissions:
-#      contents: 'read'
-#      id-token: 'write'
-#    inputs: '{ "bee-name": "${{ github.event.repository.name }}-${{ github.run_id }}-${{ matrix.terra-env }}" }'
\ No newline at end of file
+  report-to-sherlock:
+    # Report new cancerdata version to Broad DevOps
+    uses: broadinstitute/sherlock/.github/workflows/client-report-app-version.yaml@main
+    needs: [ tag-build-publish ]
+    with:
+      new-version: ${{ needs.tag-build-publish.outputs.tag }}
+      chart-name: 'cancerdata'
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+#    inputs: '{ "bee-name": "${{ github.event.repository.name }}-${{ github.run_id }}-${{ matrix.terra-env }}" }'
```

Comment:
Is there a reason to keep this comment in? NBD, just wondering from a hygiene perspective.

Reply:
I'll remove